react/features/mobile/background/middleware.native.ts

```diff
@@ -74,8 +74,11 @@ function _setAppStateListenerF({ getState }: IStore, next: Function, action: Any
     const result = next(action);
     const { appStateListener: newListener } = getState()['features/background'];
 
+    // @ts-ignore
+    const appStateOldListener = AppState.addEventListener('change', oldListener);
```

Comment:
This won't remove the previous listener. We need to store the return value of AppState.addEventListener and place that in redux.

Reply:
I created an action(eventSubcription) that stores the previous appStateListener.